roles/batmancompile/templates/dkms.conf.j2

```diff
@@ -1,3 +1,5 @@
+#jinja2: lstrip_blocks: "True", trim_blocks: "True"
```

Comment:
Das jinja2-Kommando wird nicht in der Datei ausgegeben sondern verhindert bei der Template-Generierung, dass etwaige Einzüge mit mehreren Leerzeichen unwillkürlich gekürzt werden.